Make.config

```diff
@@ -60,8 +60,8 @@ IOS_PACKAGE_UPDATE_ID=$(shell printf "2%02d%02d%02d%03d" $(IOS_PACKAGE_VERSION_M
 
 # Xcode version should have both a major and a minor version (even if the minor version is 0)
 XCODE_VERSION=12.2
-XCODE_URL=http://xamarin-storage/bot-provisioning/xcodes/Xcode_12.2_beta_3.xip
-XCODE_DEVELOPER_ROOT=/Applications/Xcode_12.2.0-beta3.app/Contents/Developer
+XCODE_URL=https://xamarin-storage/bot-provisioning/xcodes/Xcode_12.2_GM_seed.xip
```

Comment:
The reason I never added the https because the certificate cannot be validated:
```
 ERROR: certificate common name ‘synology.com’ doesn't match requested host name ‘xamarin-storage’.
```

I'd move it to http, get RE to fix it :/